aws-single-instance-resiliency/Chart.yaml

```diff
@@ -22,3 +22,22 @@ version: 41.1.0
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "3.41.1"
+
+keywords:
+  - artifacts
+  - dependency
+  - management
+  - sonatype
+  - nexus
+  - repository
+  - quickstart
+  - ci
+  - repository-manager
+  - nexus3
+home: https://www.sonatype.com/nexus-repository-oss
+icon: https://sonatype.github.io/helm3-charts/NexusRepo_Vertical.svg
```

Comment:
Should the icon be moved to this repo?

Reply:
I believe  https://sonatype.github.io/helm3-charts/NexusRepo_Vertical.svg in in the github pages site